src/components/navigation/header/config/DrawerNodeItems.tsx

```diff
@@ -294,1110 +1253,58 @@ export const drawerNodeItems = ({
-            it: `https://partnerhub.financescout24.ch/it/landing?dealerId=${sellerId}&utm_source=autoscout24.ch&utm_medium=web&utm_campaign=link_as24_cardealer_portal_headernavigation_insurance`,
-          },
-          visibilitySettings: {
-            userType: {
-              private: false,
-              professional: true,
-            },
-            brand: {
-              autoscout24: true,
-              motoscout24: false,
-            },
-          },
-        },
-        {
-          translationKey: 'header.searchMenu.insuranceComparison',
-          link: {
-            de: 'https://www.financescout24.ch/de/lp/autoversicherung-finden?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_car_',
-            en: 'https://www.financescout24.ch/de/lp/autoversicherung-finden?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_car_',
-            fr: 'https://www.financescout24.ch/fr/lp/trouver-assurance-auto?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_car_',
-            it: 'https://www.financescout24.ch/it/lp/trova-assicurazione-auto?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_car_',
-          },
-          visibilitySettings: {
-            userType: {
-              private: true,
-              professional: false,
-            },
-            brand: {
-              autoscout24: true,
-              motoscout24: false,
-            },
-          },
-        },
-        {
-          translationKey: 'header.userMenu.insuranceComparison',
-          link: {
-            de: 'https://www.financescout24.ch/de/lp/autoversicherung-finden?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_moto_',
-            en: 'https://www.financescout24.ch/de/lp/autoversicherung-finden?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_moto_',
-            fr: 'https://www.financescout24.ch/fr/lp/trouver-assurance-auto?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_moto_',
-            it: 'https://www.financescout24.ch/it/lp/trova-assicurazione-auto?utm_source=autoscout24.ch&utm_medium=web&utm_campaign=subnavigation_moto_',
-          },
-          visibilitySettings: {
-            userType: {
-              private: true,
-              professional: true,
-            },
-            brand: {
-              autoscout24: false,
-              motoscout24: true,
-            },
-          },
-        },
+        privateAutoScoutSellLinkConfig({ trackEvent }),
+        professionalSellLinkConfig({ trackEvent }),
+        privateMotoScoutSellLinkConfig({ trackEvent }),
+        estimateLinkConfig({ trackEvent }),
+        autoScoutAssureLinkConfig({ trackEvent }),
+        motoScoutAssureLinkConfig({ trackEvent }),
+        electromobilityLinkConfig({ trackEvent }),
+        magazineLinkConfig({ trackEvent }),
       ],
     },
+  ],
+  combined: [
```

Comment:
With the items extracted and a large file diff goes a bit crazy. This is where the new configuration starts